docs/getstarted/azure-core/step06.md

```diff
@@ -28,7 +33,7 @@ interface Widgets {
   createOrUpdateWidget is Operations.LongRunningResourceCreateOrUpdate<Widget>;
```

Comment:
@markcowl 
Is it OK for us to replace this example with `LongRunningResourceCreateOrReplace`?

As guideline
[⛔](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#lro-no-patch-lro) DO NOT implement PATCH as an LRO. If LRO update semantics are required, implement it using the [LRO POST action pattern](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#lro-existing-resource) .

Reply:
I updated according to Weidong's comment. I can roll back to previous comment if needed.